.github/workflows/docker-cloudstack-simulator.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Docker Image Build
+
+on:
+  push:
+    branches:
+      - main
+    tags:        
+      - 4.*
+      - 5.*
```

Comment:
do we need these? what for?

Reply:
so this will first become active after we releasde 4.18, i suppose.

Reply:
@DaanHoogland 
for official release, yes.

also yes, for each commit in "main" branch if this is merged.